python/ray/data/datasource/tensorflow_datasource.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import TYPE_CHECKING, Callable, Iterator, List
+
+from ray.data.block import Block, BlockMetadata, T
+from ray.data.datasource import Datasource, ReadTask
+
+if TYPE_CHECKING:
+    import tensorflow as tf
+
+logger = logging.getLogger(__name__)
+
+
+class SimpleTensorFlowDatasource(Datasource[T]):
```

Comment:
Similar to `SimpleTorchDatasource` (#23926), I'm not sure where to place this in the Ray Data documentation (if at all).